tests/e2e/vmservice_vm.go

```diff
@@ -98,6 +99,11 @@ var _ bool = ginkgo.Describe("[vmsvc] vm service with csi vol tests", func() {
 			dsRef.Value, &e2eVSphere)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
+		if latebinding {
+			storageClassName = storageClassName + "-latebinding"
+			framework.Logf("Late Binding StorageClassName: %s", storageClassName)
```

Comment:
Please declare it as a constant if you are going to use it again in other files as well.